spec/controllers/email_actions_controller_spec.rb

```diff
@@ -19,6 +19,24 @@
     end
   end
 
+  describe "follow_discussion" do
+    before do
+      @user = FactoryGirl.create(:user)
```

Comment:
Actually, I prefer the FactoryGirl format.. It makes it really clear where the create method comes from.